driver/driver_test.go

```diff
@@ -16,35 +17,44 @@ func TestDriver(t *testing.T) {
 	RunSpecs(t, "Contrail Network Driver test suite")
 }
 
+var _ = BeforeSuite(func() {
+	err := common.HardResetHNS()
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = AfterSuite(func() {
+	err := common.HardResetHNS()
+	Expect(err).ToNot(HaveOccurred())
+})
+
 var _ = Describe("Contrail Network Driver", func() {
 
-	contrailDriver := &driver.ContrailDriver{}
+	contrailDriver := &ContrailDriver{}
 
 	Context("upon starting", func() {
 
-		It("listens on a pipe", func() {
+		PIt("listens on a pipe", func() {
```

Comment:
PIt means "Pending", which means the test will be skipped (along with lengthy setup and teardown)